Specs/Scene/UrlTemplateImageryProviderSpec.js

```diff
@@ -146,7 +146,7 @@ defineSuite([
             expect(provider.maximumLevel).toBeUndefined();
             expect(provider.minimumLevel).toBe(0);
             expect(provider.tilingScheme).toBeInstanceOf(WebMercatorTilingScheme);
-            expect(provider.rectangle).toEqual(rectangle);
+            expect(provider.rectangle).toEqualEpsilon(rectangle, CesiumMath.EPSILON14);
```

Comment:
Before someone asks, this change was necessary because the `UrlTemplateImageryProvider` now intersects the input rectangle with the tiling scheme's rectangle, and `Rectangle.intersect` does a bunch of things that together can introduce very tiny rounding errors.